qtmodules.conf

```diff
@@ -65,150 +74,128 @@
 	branch = 5.12.4
 	status = essential
 	priority = 40
-[submodule "qtrepotools"]
```

Comment:
I removed some in here since they don't exist in the qt releases repo anymore, I just assumed it is old stuff.